guide/getting-started/12-technology-primer.md

```diff
@@ -0,0 +1,281 @@
+
+</div>
+
+## What is a block?
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/getting-started/technology/block.jpg"
+   retina = "/assets/images/guide/getting-started/technology/block@2x.jpg"
+   alt-text = ""
+   width = 400
+   height = 400
+   layout = "float-right-desktop"
+%}
+
+From the mempool, transactions are bundled into blocks. One a transaction is in a block that the network has accepted, the transaction is considered "confirmed". Blocks are created roughly every 10 minutes.
+
+They have restrictions on file size, so a block can only contain a certain number of transactions.
+
+</div>
+
+## What is hashing?
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/getting-started/technology/hashing.jpg"
+   retina = "/assets/images/guide/getting-started/technology/hashing@2x.jpg"
+   alt-text = ""
+   caption = "Example of a SHA-256 hash."
+   width = 400
+   height = 400
+   layout = "float-right-desktop"
+%}
+
+Another fundamental technique to cryptography and Bitcoin is called hashing. It is the process of taking data and applying a function that creates a unique identifier for it. This is called a hash and can be thought of as a fingerprint.
+
+This process cannot be reversed, meaning that that the original data cannot be restored from the hash. Because hashes are small in size, they are a great way to verify the authenticity of specific information. 
+
+</div>
+
+## What is a block chain?
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/getting-started/technology/blockchain.jpg"
+   retina = "/assets/images/guide/getting-started/technology/blockchain@2x.jpg"
+   alt-text = ""
+   width = 400
+   height = 400
+   layout = "float-right-desktop"
+%}
+
+A term that has become widely popular, blockchain refers to the linking of blocks by embedding information about one block into the next one. Every block contains a hash of the previous block. This allow for verifying that new blocks are based on the same data set that nodes in the Bitcoin network have agreed on.
+
+</div>
+
+## How are transactions processed?
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/getting-started/technology/mining-a-block.jpg"
+   retina = "/assets/images/guide/getting-started/technology/mining-a-block@2x.jpg"
+   alt-text = ""
+   width = 400
+   height = 400
+   layout = "float-right-desktop"
+%}
+
+This function is performed by miners, which are computers that group new transactions from the mempool into blocks and try to get those blocks accepted by the Bitcoin network. They are called miners because the creation of new blocks is rewarded by freshly minted bitcoin. This is the only way bitcoin are created, so there is heavy competition around mining.
```

Comment:
An important function of the miners is that they verify that an output can be spent by checking those in the transaction against the UTXO set.